cuegui/cuegui/ItemDelegate.py

```diff
@@ -366,3 +366,51 @@ def __init__(self, parent, *args):
 
     def paint(self, painter, option, index):
         AbstractDelegate.paint(self, painter, option, index)
+
+
+class ProgressDelegate(AbstractDelegate):
+
+    def __init__(self, parent, *args):
+        AbstractDelegate.__init__(self, parent, *args)
+
+    def paint(self, painter, option, index):
+        if index.data(QtCore.Qt.UserRole) == cuegui.Constants.TYPE_FRAME:
+            frame = self.parent().itemFromIndex(index).rpcObject
+            opts = QtWidgets.QStyleOptionProgressBar()
+            opts.rect = option.rect
+            opts.minimum = 1
+            opts.maximum = 100
+            opts.textVisible = True
+
+            if frame.data.state == opencue.api.job_pb2.SUCCEEDED:
+                progress = 100
+            elif frame.data.state == opencue.api.job_pb2.RUNNING:
+                progress = int(cuegui.Progress.progress(frame.id()))
```

Comment:
@larsbijl Question about this change -- `cuegui.Progress` on this line, what does that refer to? Is there meant to be a new class/module added in this change that is missing?

Reply:
This is a change we have in our branch where we use progress information gathered from the frame out of redis.
it must have slipped in.
 
This section on progress for the `TYPE_FRAME` can be removed as we don't have this implemented yet.